4-compartment.md

```diff
@@ -0,0 +1,834 @@
+
+# Compartment
+
+## Synopsis
+
+Compartments provide a high-level API for orchestrating modules and evaluators,
+such that programs and modules can be collectively isolated to a particular
+global scope.
+We expect that compartments will be used to isolate Node.js packages and import
+map scopes.
+
+Agoric's [SES shim][ses-shim], Sujitech's [ahead-of-time-SES][AOT-SES] and
+Moddable's [XS][xs-compartments] are actively vetting this proposal
+as a shim and native implementation respectively (2022).
+
+This proposal is a [SES Proposal][ses-proposal] milestone.
+
+## Motivation
```

Comment:
I'm confused here. What is it that you can't do in user-land with 0, 1, 2 and 3 from above that this (4) will give you?

Reply:
1. Higher-level API that would allow for terser orchestration in bundle runtimes, which are sensitive about their weight.
2. Fewer reified objects. Embedded systems will likely implement `Compartment` natively regardless of whether compartments can be implemented in user code to avoid unnecessary reification of intermediate objects. Our friends at Moddable are not quite ready to delete layer 4 from the proposal.